docs/application/dotnet/guides/nui/quickpanelclient.md

```diff
@@ -0,0 +1,101 @@
+# Quickpanel Client
+
+Quickpanel is a library to control the Quickpanel service window that shows notifications and system setup widgets. You can use Quickpanel to get various information from the Quickpanel service window. For example, media player application needs to close the Quickpanel service window during playback of video. In this case, you can use the Quickpanel client.
+
+## Prerequisites
+To use the methods of the Quickpanel API, use `Tizen.NUI.WindowSystem` in your application:
+```
+using Tizen.NUI.WindowSystem
+```
+
+## Create QuickPanel handle
+After you have created the main window of your application, call `QuickPanelClient()` with a TizenShell and Window to create the handler:
+```
+private void onInitialize()
+{
+    Window window = NUIApplication.GetDefaultWindow();
+    Shell.TizenShell tzShell = new Shell.TizenShell();
+    Shell.QuickPanelClient qpClient = new Shell.QuickPanelClient(tzShell, window, Shell.QuickPanelClient.Types.SystemDefault);
+
+    ... do nice things with qpClient ...
+}
+```
+
+## Show or Hide Quickpanel Window
+To show or hide the Quickpanel service window when your application’s window is activated, use the following code:
+```
+public void qpShowHide()
+{
+    Window window = NUIApplication.GetDefaultWindow();
+    Shell.TizenShell tzShell = new Shell.TizenShell();
+    Shell.QuickPanelClient qpClient = new Shell.QuickPanelClient(tzShell, window, Shell.QuickPanelClient.Types.SystemDefault);
+
+    qpClient.Hide();
+
+    ... doing something without quickpanel ...
+
+    qpClient.Show();
+}
+```
+
+## Get Status of Quickpanel Window
+To get the current status of the Quickpanel service window, you can get status that what you wants like below codes:
+```
+... after qpClient createad ...
+
+VisibleState visible = qpClient.Visible;
+Window.WindowOrientation orientation = qpClient.Orientation;
+ScrollableState scrollable = qpClient.Scrollable;
+```
+Each of status meaning like below.
+* **VisibleState**: the visible state of quickpanel service window. It can be Unknown, Shown, and Hidden
+* **Orientation**: the orientation of quickpanel service window. It has 4 angles, Angle_0, Angle_90, Angle_180, and Angle_270.
+* **ScrollableState**: the scrollable state of quickpanel service window. the detailed scrollable states are like below.
+    * Unknown: Unknown state
+    * Set: The quickpanel service window can scrollable(draggable)
+    * Unset: The quickpanel service window can NOT scrollable
+    * Retatin: The quickpanel service window's scrollable state will be following below visible window's scrollable state.
+
+## Set Scrollable State of Quickpanel Window
+Quickpanel window can be scrollable or not scrollable by Quickpanel Client Window's scrollable state.
```

Comment:
Not clear, please rephrase.

Reply:
Done.

Quickpanel client window can control scrollable state of the Quickpanel service window.